scripts/development/m3_stack/start_m3.sh

```diff
@@ -10,7 +10,7 @@ if [[ "$FORCE_BUILD" = true ]] ; then
 fi
 
 echo "Bringing up nodes in the background with docker compose, remember to run ./stop.sh when done"
-docker-compose -f docker-compose.yml up $DOCKER_ARGS m3coordinator01
+docker-compose -f docker-compose.yml up --build $DOCKER_ARGS m3coordinator01
```

Comment:
Intentional to have this change in here?

Reply:
Oops